src/nls/de/menu.json

```diff
@@ -91,6 +91,7 @@
         },
         "CREATE_CLIPPING_MASK": "Schnittmaske erstellen",
         "CREATE_ARTBOARD": "Neue Zeichenfläche erstellen",
+        "DUPLICATE": "Auswahl duplizieren",
```

Comment:
We already had duplicate string in EDIT, so I copied it across to Arrange as well, to save l10n some time.